src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/GetCreationTimeTests.cs

```diff
@@ -69,7 +68,7 @@ public void GetCreationTime_GetsTime_Unix()
 
                 // Filesystem timestamps vary in granularity, we can't make a positive assertion that
                 // the time will come before or after the current time.
-                Assert.Equal(default(DateTimeOffset), isf.GetCreationTime(file));
+                Assert.Equal(default(DateTimeOffset).ToLocalTime(), isf.GetCreationTime(file));
```

Comment:
The Fedora machines have time zone info set with an offset of -7:53. I would have expected this test to fail in more places because any computer that isn't using GMT as its local time would have `default(DateTimeOffset) != default(DateTimeOffset).ToLocalTime()`. I am not sure what time zone the other machines are set to.